pull_request_template.md

```diff
@@ -0,0 +1,4 @@
+Thanks for submitting your PR.
+Please note that localization contributions to any android-l10n product should be submitted through Pontoon: https://pontoon.mozilla.org/projects/android-l10n/
+
+If you are interested in localizing one of the android-l10n projects, please file an issue against the [template here](.github/ISSUE_TEMPLATE/new-localization-request.md).
```

Comment:
Is this intake pipeline recommended because people can't make new locale requests for specific android-l10n projects in Pontoon?

Reply:
Yes, that is correct. The only way they have right now is to know they need to reach out directly to me